src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -78,8 +79,15 @@
 }
 
 retry_decorator = retry(
-    on=(RequestException, ApiHostError, ApiTimeoutError),
-    on_silent=(ChunkedEncodingError),
+    on=(RequestException),
+    on_silent=(
+        ChunkedEncodingError,
+        Timeout,
+        ApiHostError,
```

Comment:
Do we expect to be able to reach the host on retries? 🤔 Are these normally blips or more long standing errors